clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -109,15 +109,28 @@ export default async function handler(
 
   const fidesString = environment.settings.FIDES_STRING;
 
-  // Check if a geolocation was provided via headers or query param
-  const geolocation = await lookupGeolocation(req);
+  let geolocation;
+  let propertyId;
 
-  const propertyId = safeLookupPropertyId(
-    req,
-    geolocation,
-    environment,
-    fidesString
-  );
+  try {
+    // Check if a geolocation was provided via headers or query param
+    geolocation = await lookupGeolocation(req);
+
+    propertyId = safeLookupPropertyId(
+      req,
+      geolocation,
+      environment,
+      fidesString
+    );
+  } catch (error) {
+    if (error instanceof Error) {
+      console.error(error);
```

Comment:
Add this line before the console.error to disable the lint error
```suggestion
      // eslint-disable-next-line no-console
      console.error(error);
```